addons/web_editor/static/src/scss/wysiwyg_snippets.scss

```diff
@@ -2965,6 +2965,10 @@ we-select.o_grid we-toggler {
                     max-width: 100%;
                 }
             }
+
+            .o_animate {
+                visibility: visible;
```

Comment:
I would also add `animation-name: none;` here. Indeed, the animation could apply an opacity or a transform that would make the animated element not visible as well.